modules/apps/headless/headless-site/headless-site-test/src/testIntegration/java/com/liferay/headless/site/resource/v1_0/test/SiteResourceTest.java

```diff
@@ -248,6 +288,85 @@ private void _testGetSiteByExternalReferenceCodeWithDollar()
 		assertValid(getSite);
 	}
 
+	private void _testGetSitesPageWithActiveOrSiteGroups(
+			boolean active, boolean site)
+		throws Exception {
+
+		Page<Site> sitesPage = siteResource.getSitesPage(
+			null, Pagination.of(1, 100));
+
+		List<Site> originalItems = (List<Site>)sitesPage.getItems();
+
+		Site postSite = testGetSitesPage_addSite(randomSite());
+
+		Group group = _groupLocalService.getGroup(postSite.getId());
+
+		group.setSite(site);
```

Comment:
@jonathanmccann, this is on purpose, setSite comes before setActive, the SF changes it